socialdistribution/socialdistribution/settings.py

```diff
@@ -23,7 +23,7 @@
 load_dotenv()
 # env variables
 SECRET_KEY = os.environ.get("SECRET_KEY")
-DEBUG = os.environ.get("DEBUG")
+DEBUG = (os.environ.get("DEBUG") == 'True')
```

Comment:
Do we want DEBUG to be true on deployment? Maybe but I thought I would ask

Reply:
the Heroku env variables have to be stored as strings, so I think this should check if the 'False' == 'True' on Heroku, so I think it should not be in debug mode when deployed